src/commands/logout.ts

```diff
@@ -20,14 +24,19 @@ export default class Logout extends Command {
 
     const params = new URLSearchParams()
     params.append("access_token", token)
-    const response = await fetch(Gravity.urls.access_tokens, {
+
+    const response = await fetch(Gravity.urls(isStaging).access_tokens, {
```

Comment:
todo: it would probably be better to just check which tokens are set and clear out both staging and prod then requiring a flag